connectivity/tests/host.go

```diff
@@ -36,24 +36,28 @@ func (s *podToHost) Run(ctx context.Context, t *check.Test) {
 			node := node
 
 			t.ForEachIPFamily(func(ipFam check.IPFamily) {
+				dstAddrs := []string{}
 				for _, addr := range node.Status.Addresses {
 					if check.GetIPFamily(addr.Address) != ipFam {
 						continue
 					}
+					dstAddrs = append(dstAddrs, addr.Address)
```

Comment:
Why this change? Let's not change any of the logic in this commit please. If you think this fixes a bug, we can change this in a separate commit or PR.

Reply:
Okay, I'll revert it.